packages/flutter/lib/src/animation/scroll_behavior.dart

```diff
@@ -90,8 +100,8 @@ class UnboundedBehavior extends ExtentScrollBehavior {
     );
   }
 
-  Simulation createSnapScrollSimulation(double startOffset, double endOffset, double velocity) {
-    return _createSnapScrollSimulation(startOffset, endOffset, velocity);
+  Simulation createSnapScrollSimulation(double startOffset, double endOffset, double startVelocity, double endVelocity) {
```

Comment:
I agree that the organization of the ScrollBehavior classes could be cleaned up. It would definitely be simpler to just have a global createSnapScrollSimulation method (perhaps in scrollable.dart, where it's used). 
https://github.com/flutter/flutter/issues/844